datastore-design/go-grpc-demo/examples/clients/python/client.py

```diff
@@ -22,7 +22,7 @@ def callAddTimeSeries(stub):
         other3='value3',
     )
     request = dstore.AddTSRequest(
-        id=1234,
+        id=1234567890,
```

Comment:
Any particular reason for this change? (asking just for curiosity!)

Reply:
It overlapped with the generated ID of the client_knmi_station.py script. 
At the moment I did not know a better way of creating an ID than a simple index/counter. The index overlaps with this one.  I increased the number so it does not overlap with the test/dummy data inserted.